sys/include/net/gnrc/netdev2.h

```diff
@@ -102,6 +103,16 @@ typedef struct gnrc_netdev2 {
      * @brief general information for the MAC protocol
      */
     uint16_t mac_info;
+
+    /**
+     * @brief device's l2 address
+     */
+    uint8_t  l2_addr[IEEE802154_LONG_ADDRESS_LEN];
```

Comment:
Just realized that, maybe what I want (for 15.4 device) in this PR is already [here](https://github.com/RIOT-OS/RIOT/blob/master/drivers/include/net/netdev2/ieee802154.h#L106)..

But, didn't understand that, for example, in [netdev2_ieee802154_set](https://github.com/RIOT-OS/RIOT/blob/master/drivers/netdev2_ieee802154/netdev2_ieee802154.c#L135), why we don't call `dev->netdev.driver->set` in [netdev2_ieee802154_set](https://github.com/RIOT-OS/RIOT/blob/master/drivers/netdev2_ieee802154/netdev2_ieee802154.c#L153) to update the address at lower level at the same time?